### ISSUE TITLE
Darwin AMD64 Cross Compiled Binary Issue

### ISSUE THREAD

**Issue by MattSurabian** [MattSurabian/pact#9]
Need repro to determine if it's the actual compilation, a non-cross platform friendly piece of code, or a makefile packaging error.

Known:
- Windows binary tested on a VM confirmed working.

**Comment by MattSurabian** [comment]
Confirmed one can compile via `make` and produce a functional binary for darwin/amd64 ON darwin/amd64.

**Comment by MattSurabian** [comment]
Error on the cross-compiled binary:

```
 ./pact 
panic: user: Current not implemented on darwin/amd64

goroutine 1 [running]:
main.init.1()
/home/matt/Code/go/src/github.com/mattsurabian/pact/config.go:43 +0x73
main.init()
/home/matt/Code/go/src/github.com/mattsurabian/pact/version.go:3 +0x1aa
```

**Comment by MattSurabian** [comment]
Replacing released binary with the compiled version made on darwin/amd64 while this gets triage'd. Removing the i386 build assuming it's also suffering the same issue.

**Comment by MattSurabian** [comment]
Hashimoto to the rescue: https://github.com/mitchellh/go-homedir via https://github.com/ncw/rclone/issues/15

**Comment by MattSurabian** [comment]
Resolved with #10 and update released.
